cmd/example/main.go

```diff
@@ -17,8 +17,15 @@ func main() {
 	// The public key above match the signature of the below file served by our CDN
 	httpSource := selfupdate.NewHTTPSource(nil, "http://geoffrey-test-artefacts.fynelabs.com/nomad.exe")
 	config := &selfupdate.Config{
-		Source:    httpSource,
-		Schedule:  selfupdate.Schedule{FetchOnStart: true, Interval: time.Minute * time.Duration(60)},
+		Source: httpSource,
+		Schedule: selfupdate.Schedule{
+			FetchOnStart: true,
+			Interval:     time.Minute * time.Duration(60),
```

Comment:
I don't know if this is a good example - as the "ScheduleAt" will happen in addition to every hour after start...